iceoryx_hoofs/README.md

```diff
@@ -99,23 +99,23 @@ The module structure is a logical grouping. It is replicated for `concurrent` an
 
 | class                              | internal | description                                                                                                                                                                                                                        |
 |:----------------------------------:|:--------:|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-|`FiFo`                              | i        | Single producer, single consumer lock-free FiFo                                                                                                                                                                                    |
-|`LockfreeQueue`                     |          | Multi producer, multi consumer lock-free FiFo with ringbuffer like overflow handling                                                                                                                                               |
-|`LoFFLi`                            | i        | Lock-free LIFO based index manager (lock-free free list). One building block of our memory manager. After construction it contains the indices {0 ... n} which you can acquire and release.                                        |
-|`SoFi`                              | i        | Single producer, single consumer lock-free safely overflowing FiFo (SoFi).                                                                                                                                                         |
-|`ResizeableLockFreeQueue`           |          | Resizeable variant of the `LockfreeQueue`                                                                                                                                                                                          |
+|`SpscFifo`                          | i        | Single producer, single consumer lock-free FiFo                                                                                                                                                                                    |
```

Comment:
I would prefer for all constructs a separate namespace describing the concurrency properties, so a namespace called `spsc` and `mpmc` and then leave the names as they were before.

Reply:
So you would like to have a `mpmc` namespace instead of a `concurrent` namespace?

Just for clarification, I started to do this due to the includes. Since `FIFO` is such a generic name I wanted to prevent name clashes and renamed the file to `spsc_fifo.hpp`. Everything else was a consequence of this renaming

Reply:
We can do this also at a later point, it was just my personal taste.

My personal favorite would be:
* `iox::lock_free::{mpmc|spsc}` - for all lockfree code
* `iox::concurrent` for generic thread safe code

But I would suggest that we talk about it in a later meetup and go from there.

Reply:
I'll add this as task to the issue

Reply:
Added to #1593